hudi-utilities/src/main/java/org/apache/hudi/utilities/schema/processor/SchemaPostProcessor.java

```diff
@@ -16,11 +16,12 @@
  * limitations under the License.
  */
 
-package org.apache.hudi.utilities.schema;
+package org.apache.hudi.utilities.schema.processor;
```

Comment:
This class is a public class. unless we have strong reasons, would prefer to keep it that way. Just incase someone has their own impl out there in the open (it may break when they upgrade)